app/javascript/app.html

```diff
@@ -1,46 +1,60 @@
 <div id="verification-tool">
-  <div v-if="loaded && statements.length === 0">
+  <div v-if="loaded && currentStatements.length === 0">
     No statements
   </div>
-  <div v-if="loaded && statements.length > 0">
-    <table style="width: 100%; border: 1px solid #ccc; border-collapse: collapse; table-layout: fixed;">
+  <div v-if="loaded && currentStatements.length > 0">
+    <button v-on:click="prevStatement()" class="mw-ui-button">Previous</button>
+    <button v-on:click="nextStatement()" class="mw-ui-button">Next</button>
+    <span style="float: right;">
+      Type
+      <select v-model="displayType">
+        <option value='all'>Show all</option>
+        <option>verifiable</option>
+        <option>unverifiable</option>
+        <option>reconcilable</option>
+        <option>actionable</option>
+        <option>manually_actionable</option>
+        <option>done</option>
+      </select>
+      &nbsp;&nbsp;
+      Statement
+      <input v-model:number="displayIndex" type="number" style="width: 40px;">
+      of {{ currentStatements.length }}
+    </span>
```

Comment:
I think these controls need to be moved to outside their parent `<div>`. Otherwise when you select an option from the dropdown with no matching statements the controls disappear, because of the `currentStatements.length > 0` check.

Reply:
Good catch, have pushed a new commit